qe_metrics/libs/database.py

```diff
@@ -1,20 +1,115 @@
+import os
+from types import TracebackType
+from typing import Optional, Type
+
+import click
+from datetime import date
+from pony import orm
 from pyaml_env import parse_config
 from simple_logger.logger import get_logger
 
+DB_CONNECTION = orm.Database()
```

Comment:
should be under `Database`

Reply:
@rnetser This needs to be accessed by the `Services` and `JiraIssues` class. If this value is under the `Database` class, as a class variable that can still be accessed by those classes, `NameError: name 'Database' is not defined` occurs because the `Database` class is still being defined when the those classes are trying to access that variable.

The other option is to have the `Services` and `JiraIssues` classes defined outside the `Database` class, as they were before.